scripts/synthetix/facilicate_trade.js

```diff
@@ -0,0 +1,97 @@
+const { getExchange } = require("../utils/trading_strategy_helpers")(web3, artifacts)
+const { SynthetixJs } = require("synthetix-js")
+const { calculateBuyAndSellAmountsFromPrice, fetchTokenInfoFromExchange } = require("../utils/trading_strategy_helpers")(web3, artifacts)
+
+// TODO - put this somewhere more generic. Copied from dex-contracts
+const sendTxAndGetReturnValue = async function (method, ...args) {
+  const result = await method.call(...args)
+  await method(...args)
+  return result
+}
+
+// truffle uses network by name and SynthetixJS uses network by ID
+const networkMap = {
+  "mainnet": 1,
+  "rinkeby": 4,
+}
+
+const argv = require("yargs")
+  .option("spread", {
+    type: "float",
+    describe: "Percentage increase required for trade (fees not accounted)",
```

Comment:
What exactly do you mean by fees not accounted? Don't our limit orders include the Gnosis protocol trading fee?

Reply:
This text was copy-pasted from the `place_spread_orders` script in `dex-contracts` and written before I had realized that fees were embedded directly into the orders themselves. Will change this text or possibly just make all the configuration parameters part of the containerized environment. WDYT?

Reply:
Actually, looking at a comment from below, I will attempt to fetch the spread form the synthetix platform via smart contract call. Zwei Vogel mit Ein Stein.